python/n.py

```diff
@@ -33,6 +30,11 @@
 with open(filename, "r") as f:
 	file = File(f)
 
+global_scope = Scope(relative_path="/".join(filename.replace("\\", "/").split("/")[0:-1]) + "/")
```

Comment:
fancy windows user; does Python not have its own built-in cross-platform file path resolver?